src/azurelinux/3.0/net10.0/source-build-test/amd64/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/sdk:10.0-preview-azurelinux3.0-amd64
+
+# Install scancode
+# Install instructions: https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#installation-as-a-library-via-pip
+# See latest release at https://github.com/nexB/scancode-toolkit/releases
+ENV venv="/tmp/scancode-env"
+RUN SCANCODE_VERSION="32.3.3" && \
+    #!/bin/bash
+    set -euo pipefail && \
+    python3 -m venv $venv && \
+    source $venv/bin/activate && \
+    pip install scancode-toolkit==$SCANCODE_VERSION && \
+    deactivate
+
+# Setup a script which executes scancode in the virtual environment
+RUN cat > /usr/local/bin/scancode <<EOF
```

Comment:
Thoughts on checking this in next to the Dockerfile and copying it in?  The benefit is improved maintainability.

Reply:
Update: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1408/commits/4db806ca9dcbb77cc4f3fc408b2fb6bf36cb7fbf